examples/nirfsg/attenuation-tables-generation.py

```diff
@@ -0,0 +1,157 @@
+r"""Download S-parameters from an attenuation per frequency table.
+
+The code shows how to convert attenuation into simple s-parameter table to be used with RFSG.
+
+The gRPC API is built from the C API. NI-RFSG documentation is installed with the driver at:
+  C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niRFSG\documentation\English\RFSG.chm
+
+Getting Started:
+
+To run this example, install "NI-RFSG Driver" on the server machine:
+  https://www.ni.com/en-us/support/downloads/drivers/download.ni-rfsg.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI-RFSG gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-RFSG-C-Function-Reference
+
+Running from command line:
+
+Server machine's IP address, port number, and physical channel name can be passed as separate
+command line arguments.
+  > python getting-started-single-tone-generation.py <server_address> <port_number> <resource_name>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763", with "SimulatedRFSG" as the physical channel name.
+"""  # noqa: W505
+
+import math
+import sys
+
+import grpc
+import nidevice_pb2 as nidevice_grpc
+import nirfsg_pb2 as nirfsg_types
+import nirfsg_pb2_grpc as grpc_nirfsg
+import numpy as np
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+SESSION_NAME = "NI-RFSG-Session"
+
+# Resource name, channel name and options for a simulated 5652 client.
+RESOURCE = "5840_1"
+OPTIONS = ""
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+if len(sys.argv) >= 4:
+    RESOURCE = sys.argv[3]
+    OPTIONS = ""
+# Create a gRPC channel + client.
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}")
+client = grpc_nirfsg.NiRFSGStub(channel)
+vi = None
+
+
+def raise_if_error(response):
+    """Raise an exception if an error was returned."""
+    if response.status != 0:
+        response = client.ErrorMessage(
+            nirfsg_types.ErrorMessageRequest(error_code=response.status)
+        )
```

Comment:
```suggestion
        response = client.ErrorMessage(nirfsg_types.ErrorMessageRequest(error_code=response.status))
```
The `black` formatter is complaining about this line in the CI's validate_python step and is expecting it to be on a single line.